src/Html/HtmlFormatter.php

```diff
@@ -472,7 +477,7 @@ protected function FormatText($text)
 
   protected function GetSourceEncoding()
   {
-    if (isset($this->state->font)) {
+    if (!is_null($this->state->font)) {
```

Comment:
Does this need a `?->` optional chain now? isset silences the warning but is_null will raise a warning if `$this->state` is null.